pkg/registration/get_kubeconfig.go

```diff
@@ -0,0 +1,79 @@
+package registration
```

Comment:
question: Do we really need get_ as prefix for this file? This file seems to do quite a lot of not getting also.